scripts/build-server

```diff
@@ -21,7 +21,8 @@ CGO_ENABLED=0 go build -tags k8s \
   -o bin/rancher
 
 if  [ -n "$CATTLE_KDM_BRANCH" ]; then
```

Comment:
Shouldn't we be setting `CATTLE_KDM_BRANCH` so there is no confusing what is used? This is still set in multiple places to a different value.

Reply:
We still want Rancher to start with branch `release-v2.6` in rke-metadata-settings, this is set by our setting logic by reading `CATTLE_KDM_BRANCH` so I didn't change it. 
https://github.com/rancher/rancher/blob/release/v2.6/pkg/settings/setting.go#L244 
https://github.com/rancher/rancher/blob/release/v2.6/pkg/settings/setting.go#L250